src/main/java/com/google/api/codegen/discovery/MainDiscoveryProviderFactory.java

```diff
@@ -61,10 +61,12 @@
       SAMPLE_METHOD_TO_VIEW_TRANSFORMER_MAP =
           ImmutableMap.of(
               JAVA, JavaSampleMethodToViewTransformer.class,
+              GO, GoSampleMethodToViewTransformer.class,
```

Comment:
I didn't have one, I think it was based on the order of the if statements that was already there.

I've ordered it alphabetically now.